app/views/hiring_staff/sign_in/email/sessions/choose_organisation.html.haml

```diff
@@ -0,0 +1,20 @@
+- content_for :page_title_prefix do
```

Comment:
same as before, just seems cleaner on one line to me so I've been changing them when I see them